db/migrate/20200114095428_create_whitehall_migration_timeline_entries.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateWhitehallMigrationTimelineEntries < ActiveRecord::Migration[6.0]
+  def change
+    create_table :whitehall_migration_timeline_entries do |t|
+      t.string "entry_type"
+      t.text "body"
```

Comment:
I don't think we should add this body column in straight away as I'm a bit worried this will solve our problem for notes but then leave us wondering what to do for fact check which will have richer data (I think I suggested in the card that we could have a JSON column). So I'd suggest maybe the limit of this PR is to get the types in and then think about how the different extra data will be represented.

Reply:
I've removed the `body` column.  We can introduce whatever we decide on in the later stories (fact check and editorial remarks).